src/partner/ducks/eventOrders.ts

```diff
@@ -0,0 +1,180 @@
+import { IOrder } from "../models/Order";
+import { EventService } from "../services";
+
+export const MODULE = "partner/eventOrders/";
+export const FETCH_EVENT_ORDERS_START = `${MODULE}/FETCH_EVENT_ORDERS_START`;
```

Comment:
rename to `FETCH_ALL_ORDERS_BY_EVENT_ID_START`, `FETCH_ALL_ORDERS_BY_EVENT_ID_SUCCESS` and `FETCH_ALL_ORDERS_BY_ID_FAIL`